changelogs/2024-08-15_enable-mandatory-sso-or-mfa-for-enterprise-plans.mdx

```diff
@@ -0,0 +1,28 @@
+---
+_schema: default
+title: Enable mandatory SSO or MFA for Enterprise Plans
+---
+This release added the option to enable mandatory SSO or MFA on Enterprise Plans and Annual payment for Lite Plans. It also updated Markdown configuration options, the Hugo and Node versions, and build image settings.
```

Comment:
> This release added the option to enable mandatory SSO or MFA on Enterprise Plans and Annual payment for Lite Plans.

Potentially some syntactic ambiguity here - can we enable enforced SSO on Lite plans?

Reply:
```suggestion
This release added the option for Annual payment for Lite Plans and and option to enable mandatory SSO or MFA on Enterprise Plans and . It also updated Markdown configuration options, the Hugo and Node versions, and build image settings.
```